src/e2e-test/features/pipeline.hierarchy.feature

```diff
@@ -21,6 +21,7 @@ Feature: Pipeline Hierarchy - Hierarchy Widgets
   Scenario: Render file and duplicate record
     Given Open Datafusion Project to configure pipeline
     Then Add "File" node to canvas
+    Then Close "Source" panel
```

Comment:
I need to do this because the side panel accordian is expanding too long. Do you know how to make it fixed height and overflow-y, like the original one? @seanfeld

Reply:
What do you mean it's expanding too long? its expanding passed the boundaries? if its covering the footer area scrolling that means something was introduced that made it do that.

The old version had it so that only one accordion could be open at once - which is a behavior we don't want - so if that is what you're seeing that is expected behavior.